pkg/cmd/controller/pipeline/pipelinerunner_cmd.go

```diff
@@ -105,7 +105,7 @@ func (o *PipelineRunnerOptions) Run() error {
 		return err
 	}
 
-	metapipelineClient, err := metapipeline.NewMetaPipelineClient()
+	metapipelineClient, err := metapipeline.NewMetaPipelineClient(o.Git(), o.In, o.Out, o.Err)
```

Comment:
I've seen this now at several places that we pass all three standard file handles explicitly. I've been thinking it would make sense to wrap this into some utility struct like `IOFileHandles`. Definitely would help to reduce the number of function parameters and I think there is potential for some shared behaviour in this struct as well then (maybe not in this case).